src/org/thoughtcrime/securesms/audio/AudioSlidePlayer.java

```diff
@@ -163,6 +172,11 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
               mediaPlayer = null;
             }
 
+            activity = getActivity(context);
+            if (activity != null) {
+              activity.getWindow().clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
```

Comment:
this is not called when pressing "pause" manually.

ftr, when leaving the Activity, things are fine, as the window with the flag set is reset.